Removed predix specific functionality and tests

irinaepshteyn opened the pull request:
- Removed predix maven profiles and service startup scripts;
- Removed predix dependencies and cloud configurations;
- Removed ZAC monitoring, predix audit, metering, and dynamic log configurer;
- Removed dependency on cloud foundry utilities for testing;
- Removed predix specific integration tests;

Commit:
US184398: Removed predix specific functionality and tests
- Removed predix maven profiles and service startup scripts;
- Removed predix dependencies and cloud configurations;
- Removed ZAC monitoring, predix audit, metering, and dynamic log configurer;
- Removed dependency on cloud foundry utilities for testing;
- Removed predix specific integration tests;

sanjeevchopra commented:
@anubhavi25 - please review this PR.  I wont get to this today.

anubhavi25 commented on README.md:
````diff
@@ -6,31 +6,46 @@ https://www.predix.io/docs#IGyNp2eM
+To build ACS without support for hierarchical attributes and run unit tests
+
+```
+mvn clean install [-P "public-titan" -s <maven_settings_file>]
````
I think you meant `[-P public-titan] [-s <maven_settings_file>]` otherwise what is being said is that both options are optional *only when specified together* instead of individually

anubhavi25 commented on acs-integration-tests/pom.xml:
```diff
@@ -284,576 +292,26 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
             <build>
```
Some common `configuration` parameters like `argLine`, `skipTests`, and `parallel` can be pulled up. However, child modules in the `predix-master` branch that run Predix-related integration tests might be included in the `<modules>` section of this POM, in which case they might erroneously inherit common configuration properties that don't pertain to them.

Here's a sample hierarchy that we might have in the future demonstrating why this might cause an issue:
```
acs-integration-tests/pom.xml
|- predix-local/pom.xml
|- predix-local-titan/pom.xml
|- predix-cloud/pom.xml
|- predix-cloud-titan/pom.xml
```

anubhavi25 reviewed (changes requested):
Sans some minor issues, looks good for the most part 👍

irinaepshteyn commented:
@anubhavi25, addressed PR comments.  As to integration tests pom refactoring, I created a separate story to consider creating modules for each test profile.
